jux/map_generator/generator.py

```diff
@@ -24,10 +24,14 @@ class GameMap(NamedTuple):
 
     @property
     def width(self) -> int:
+        if len(self.rubble.shape) == 3:
+            return self.rubble.shape[2]
         return self.rubble.shape[1]
```

Comment:
I think a more general way is to access dimensions by negative index, because the batch dimensions are always leading dimension.
```python
    @property
    def width(self) -> int:
        return self.rubble.shape[-1]

    @property
    def height(self) -> int:
        return self.rubble.shape[-2]
```
There may be more than one batch dimension.